static/css/styles.css

```
@@ -0,0 +1,803 @@
+}
+
+@media (min-width: 577px) {
+  .nav-tabs .nav-link {
+    padding: var(--spacing-sm) 0.75rem;
+    font-size: var(--font-size-sm);
+  }
+}
+
+#built-with-list .nav-link.nav-underline {
+  position: relative;
+  font-family: "YoMamaHeader";
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)